10x/count/count.wdl

```diff
@@ -987,9 +1044,9 @@ task mark_duplicates_main {
 
   runtime {
     docker: "marcusczi/cellranger_clean:cromwell"
-    memory: "30 GB"
-    cpu: 8
-    disks: "local-disk 1000 HDD"
+    memory: "33.75 GB"
```

Comment:
This feels very specific. What does 33.75 correspond to?

Reply:
A typo! It should be 3.75. Thanks.